### PR TITLE
React Native JavaScript template

### PR DESCRIPTION
**Pull request by zepedr0**
## What, How & Why?

React Native users often use a template to get started with a new app. We should provide a Realm template which helps our users to use best practices from the very beginning of the development cycle.

## ☑️ ToDos

* [ ] 📝 Changelog entry
* ~[ ] 📝 `Compatibility` label is updated or copied from previous entry~
* ~[ ] 🚦 Tests~
* [ ] 📝 Public documentation PR created or is not necessary
* ~[ ] 💥 `Breaking` label has been applied or is not necessary~

*If this PR adds or changes public API's:*
* ~[ ] typescript definitions file is updated~
* ~[ ] jsdoc files updated~
* ~[ ] Chrome debug API is updated if API is available on React Native~

### CODE REVIEW

**Review by takameyer — APPROVED** [review]
Looks good to me.

File: packages/react-native-template-realm-js/package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  "version": "0.0.1",
+  "description": "React Native Template Realm JS",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/realm/realm-js.git",
+    "directory": "packages/react-native-template-realm-js"
+  },
+  "author": {
+    "name": "Realm",
+    "email": "help@realm.io",
```
Comment (bmunkholm) [inline packages/react-native-template-realm-js/package.json:12]:
```suggestion
    "email": "realm-help@mongodb.com",
```

File: packages/react-native-template-realm-js/package.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "url": "https://realm.io"
+  },
+  "license": "MIT"
```
Comment (bmunkholm) [inline packages/react-native-template-realm-js/package.json:15]:
```suggestion
  "license": "Apache-2"
```

File: packages/react-native-template-realm-js/package.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "react-native-template-realm-js",
+  "version": "0.0.1",
```
Comment (bmunkholm) [inline packages/react-native-template-realm-js/package.json:3]:
```suggestion
  "version": "1.0.0",
```

File: packages/react-native-template-realm-js/template/package.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "realmjstemplate",
+  "version": "0.0.1",
```
Comment (bmunkholm) [inline packages/react-native-template-realm-js/template/package.json:3]:
```suggestion
  "version": "1.0.0",
```

File: packages/react-native-template-realm-js/package.json (hunk 2 above)
Comment (kneth) [reply packages/react-native-template-realm-js/package.json:15]: `Apache-2.0` complies to the [SPDX](https://spdx.org/licenses/) format

Comment (kneth) [reply packages/react-native-template-realm-js/package.json:15]: Moreover, please add https://www.apache.org/licenses/LICENSE-2.0.txt as `LICENSE`

**Review by kneth — APPROVED** [review]
The comments about what the Realm calls are doing are very good to nice.

**Pull request merged**